### PR TITLE
added file info to readme

### PR DESCRIPTION
**Pull request by ChaoticBlack**
Added the description of 0_Dataset.ipynb and 1_Preliminaries.ipynb to readme under the heading of File descriptions.

### CODE REVIEW

File: project_2_image_captioning_project/README.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ Here are some predictions from my model.
 ![sample_107](samples/sample_107.png?raw=true)<br/>
 ![sample_202](samples/sample_202.png?raw=true)
 
+## File descriptions
+**0_Dataset.ipynb :** - In this file we initialize the COCO API so that we can access the data in the Microsoft Common Objects in COntext (MS COCO) dataset.
+
```
Comment (vidhi-mody) [inline project_2_image_captioning_project/README.md:48]: Fix the minor typo in line 44

**Review by vidhi-mody — CHANGES_REQUESTED** [review]
This is a good start. If you can, add a few resources which could help people reading the file understand the .ipynb files better. I am not sure why there are so many files changed in this PR. Please remove the files that do not relate to this issue

**Comment by ChaoticBlack** [comment]
Ok I'll make the changes!
Even I didn't understand why there are so many files. I guess it has something to do with ImgBot.
I'll figure it out!

**Comment by vidhi-mody** [comment]
Let me know if you need any help with this

**Comment by ChaoticBlack** [comment]
Yes, Thank you!

**Pull request closed without merging**